src/api/entitlements.js

```diff
@@ -263,4 +288,16 @@ export class Entitlement {
       : [json];
     return jsonList.map(json => Entitlement.parseFromJson(json));
   }
+
+  /**
+   * Returns the SKU associated with this entitlement.
+   * @return {?string}
+   */
+  getSku() {
```

Comment:
A very similar method already exists. Please DRY.

https://github.com/subscriptions-project/swg-js/blob/master/src/runtime/pay-flow.js#L484

Reply:
The code is effectively the same but it is accepting different kinds of inputs.  One is a string associated with the purchase and the other is one associated with an entitlement.  Is there anything insisting these 2 things match or is it just happenstance?

I could create a generic utility function to read a single property out of a string that might be a json but it seems fairly unnecessary.

Reply:
I went ahead and centralized this logic